src/components/LightUpChinatown/DonationTierItem.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import {
+  tabletScreens,
+  phoneScreens,
+} from '../../utilities/general/responsive';
+
+interface Props {
+  imageUrl: string;
+  ctaText: string;
+  titleText: string;
+  descriptionItems: string[];
+  buttonText: string;
+  currentDonationCount: number;
+  maxDonationCount: number;
+}
+
+const DonationTierItem = (props: Props) => {
+  const { t } = useTranslation();
+  const remainingDonationCount =
+    props.maxDonationCount - props.currentDonationCount;
+
+  return (
+    <React.Fragment>
+      <Container>
+        <ColumnContainer>
+          <ImageContainer>
+            <Image src={props.imageUrl} alt="donation_tier_image" />
+          </ImageContainer>
+        </ColumnContainer>
+        <ColumnContainer>
+          <CTA>{props.ctaText}</CTA>
+          <Title>{props.titleText}</Title>
+          {props.descriptionItems.map((descriptionItem: any) => (
+            <Description>{descriptionItem}</Description>
+          ))}
+          {/* Spacer to keep text aligned between donations items */}
+          <div style={{ width: '600px' }}></div>
```

Comment:
you could probably do this better in css, like 
`ColumnContainer = styled.div'
 display: flex;
 justify-content: space-between;
'
`